packages/paste-core/components/badge/stories/index.stories.tsx

```diff
@@ -0,0 +1,246 @@
+import * as React from 'react';
+import {useUIDSeed} from 'react-uid';
+import {Stack} from '@twilio-paste/stack';
+import type {IconSizeOptions} from '@twilio-paste/style-props';
+import {InformationIcon} from '@twilio-paste/icons/esm/InformationIcon';
+import {WarningIcon} from '@twilio-paste/icons/esm/WarningIcon';
+import {SuccessIcon} from '@twilio-paste/icons/esm/SuccessIcon';
+import {ErrorIcon} from '@twilio-paste/icons/esm/ErrorIcon';
+import {NewIcon} from '@twilio-paste/icons/esm/NewIcon';
+
+import {Badge} from '../src';
+
+const ICON_SIZE_OPTIONS: IconSizeOptions[] = [
+  'sizeIcon10',
+  'sizeIcon20',
+  'sizeIcon30',
+  'sizeIcon40',
+  'sizeIcon50',
+  'sizeIcon60',
+  'sizeIcon70',
+  'sizeIcon80',
+  'sizeIcon90',
+  'sizeIcon100',
+  'sizeIcon110',
+];
+
+// Do we not want to include controls..?
```

Comment:
While we could have 1 story with controls (I think we're still using knobs even), we use Storybook for VRT so we need to have 1 story per view for testing.

Reply:
> ... we use Storybook for VRT so we need to have 1 story per view for testing.

In the most recent iteration I am moving the individual variants to their own page, though within each variant page there are three different renders. Does that align with our needs for VTR?